src/index.ts

```diff
@@ -55,7 +55,7 @@ export class AsyncQueue {
 	 * Resolving the next promise
 	 * @returns {boolean} If there is a next promise, it will return `true`, if there is no, `false`
 	 */
-	public next(): boolean {
+	public next(): Boolean {
```

Comment:
why not `boolean`?

Reply:
More of a preference of mine, Boolean is an object and boolean is a primitive type. Usually gotten issues with `boolean` from compilers, but probably fixed, so just something I do automatic when I see it.

Reply:
ok